src/index.js

```diff
@@ -1,4 +1,4 @@
 import { h, render } from 'preact';
-import JustNotSorry from "./components/JustNotSorry";
+import JustNotSorry from './components/JustNotSorry';
 
-render(<JustNotSorry />, document.body);
+render(<JustNotSorry onEvents={['input', 'focus', 'cut']} />, document.body);
```

Comment:
I think abstracting these events help address my earlier comments
https://github.com/defmethodinc/just-not-sorry/pull/174#discussion_r1088401963
in order to make testing much easier

Reply:
This is a nice improvement.  Adding the prop to inject the events makes it clearer what triggers JustNotSorry to do its thing.